.github/workflows/issues.yml

```diff
@@ -39,7 +39,7 @@ jobs:
                 "description": "This ticket tracks the following GitHub issue: '"${ISSUE_URL}"'.",
                 "components": [
                     {
-                        "id": "32225"
```

Comment:
[q] What is this id change?

Reply:
this one got pulled in since we haven't rebased from master branch. more details on this specific update in PR #1178